packages/offix-datastore/src/replication/GraphQLReplicator.ts

```diff
@@ -33,35 +34,55 @@ export interface ReplicatorMutations {
  * GraphQL mutations for create, update and delete
  */
 export interface ReplicatorSubscriptions {
-  /**
-   * GraphQL create subscription document.
-   */
-  new: string | DocumentNode;
-
-  /**
-   * GraphQL update subscription document.
-   */
-  updated: string | DocumentNode;
-
-  /**
-   * GraphQL delete subscription document.
-   */
-  deleted: string | DocumentNode;
+    /**
+     * GraphQL create subscription document.
+     */
+    new: string | DocumentNode;
+
+    /**
+     * GraphQL update subscription document.
+     */
+    updated: string | DocumentNode;
+
+    /**
+     * GraphQL delete subscription document.
+     */
+    deleted: string | DocumentNode;
+}
+
+// TODO add predicateToFilter
+interface ReplicatorQuery {
```

Comment:
This seems redundant - What would be the reason for this interface (and one bellow)

Reply:
Response formats for queries vary. CRUD Spec uses `result.items`. I'm using this interface to avoid forcing Crud Spec in the `GraphqlReplicator` layer. The one below is only for sync queries. It was just easier to define its own interface as it has a lastSync response we want to access, not just items